### DISCUSSION TITLE
Cannot receive Auth code as links not working

### DISCUSSION THREAD

**Discussion by danielmassey94** [potmat/homebridge-google-nest-sdm#169] [Q&A]
**Have you read the [FAQ](https://github.com/potmat/homebridge-google-nest-sdm#faq)?**

Yes

**Have you searched [existing issues](https://github.com/potmat/homebridge-google-nest-sdm/issues?q=is%3Aissue)?**

Yes

**Have you searched [existing discussions](https://github.com/potmat/homebridge-google-nest-sdm/discussions?discussions_q=)?**

Yes

**Describe the bug**
Using the new link provided in the description to receive an auth code brings up an error message "Something went wrong. Please contact the developer of this app if the issue persists." 

I am using link: 
https://nestservices.google.com/partnerconnections/project-id/auth?redirect_uri=https://www.google.com&access_type=offline&prompt=consent&client_id=oauth2-client-id&response_type=code&scope=https://www.googleapis.com/auth/sdm.service

**Attach Logs**
<img width="452" alt="Screenshot 2024-05-02 at 12 36 56" src="https://github.com/potmat/homebridge-google-nest-sdm/assets/168735895/687526ae-9933-47dd-972e-d3b83250d552">

**Comment by potmat** [comment]
Converted to discussion as this is not an issue with the plugin itself.
